src/charts/axis.js

```diff
@@ -58,10 +58,24 @@ export default function Axis (_container) {
     yAxis: null,
     y2Axis: null,
     horizontalGridLines: null,
-    verticalGridLines: null
+    verticalGridLines: null,
+    xLabelsShouldRotate: false
   }
 
-  let xLabelsShouldRotate = false
+  const APPROX_FONT_WIDTH = 5
+
+  const X_TICK_LABEL_SETTINGS = {
```

Comment:
@biovisualize I created a constant for the magic numbers. I think this will make it easier to set these values dynamically later on if we need to (e.g. allowing a user to set the default font size for things like labels).